templates/nsq/0/docker-compose.yml.tpl

```diff
@@ -0,0 +1,43 @@
+nsqlookupd01:
```

Comment:
If you are not using any golang templating function, please, rename `templates/nsq/0/docker-compose.yml.tpl` to `templates/nsq/0/docker-compose.yml`

Please, use docker-compose version 2  format. 
```
-nsqlookupd01:
+version: '2'
+services:
+  nsqlookupd01:
.... Indent all services accordingly.
```